java/src/Ice/src/main/java/com/zeroc/Ice/_ObjectPrxI.java

```diff
@@ -341,15 +350,6 @@ public void _write(OutputStream os) {
     _reference.streamWrite(os);
   }
 
-  @Override
-  public void _copyFrom(ObjectPrx p) {
```

Comment:
This was a hack we used to get around not being to call a constructor.
We'd construct an object using reflection, then use this to 'patch' it's fields.

This logic was kept-as-is. It's just done via a constructor now.